packages/airnode-admin/src/implementation.ts

```diff
@@ -30,7 +30,7 @@ export const deriveWalletPathFromSponsorAddress = (sponsorAddress: string): stri
     const shiftedSponsorAddressBN = sponsorAddressBN.shr(31 * i);
     paths.push(shiftedSponsorAddressBN.mask(31).toString());
   }
-  return `0/${paths.join('/')}`;
```

Comment:
Basically same comment I left here: https://github.com/api3dao/airnode-stress-tester/pull/5#discussion_r786139114

Reply:
I'd rather postpone that and revisit it properly by moving this function to a separate package or when the use case for parameterizing this occurs.

Reply:
👍🏻